docs/getting-started.md

```diff
@@ -0,0 +1,108 @@
+choose depends on your previous knowledge about programming, and how much of
+your game you want to create yourself.
+
+### High level development
+
+This is ideal if you just want to get a game done, without worrying too much about how the hardware
+works. Unfortunately, there is nothing similar to PC engines like Unity, Godot, and the GBA doesn't
+support languages like C#, Python or Jave.
+
+Our recommendation is to use [butano](https://github.com/GValiente/butano), a C++ library that
+simplifies GBA development a lot. It has become quite popular, so you will easily get support for
+it. You should follow the [getting started](https://gvaliente.github.io/butano/getting_started.html)
+instructions. It comes with a lot of examples of how to use all its features, check them out!
+
+### Low level development
+
+If you're comfortable writing to I/O registers and reading low level documentation like
+[GBATEK](https://www.problemkaputt.de/gbatek.htm) there are a few options:
+
+#### C and C++
+
+The most popular option is to use devkitARM and [libtonc](https://github.com/gbadev-org/libtonc).
+Follow the instructions in the [getting started guide](https://devkitpro.org/wiki/Getting_Started)
+to install devkitARM and check the [examples](https://github.com/gbadev-org/libtonc-examples) and
+the [template](https://github.com/gbadev-org/libtonc-template). Once you have verified that you can
+compile the examples read the tutorial [Tonc](https://gbadev.net/tonc/toc.html) (by the author of
+libtonc) to understand how GBA development goes.
+
+libgba is another library provided by devkitPro, but it isn't as recommended as using libtonc.
+
+If you don't want to use devkitARM, there are some alternatives. For example, check
+[gba-toolchain](https://github.com/felixjones/gba-toolchain), which also supports libtonc and
+libgba.
+
+If you want to use a completely different library than libtonc or libgba, you can try
+[sdk-seven](https://github.com/LunarLambda/sdk-seven).
+
+Some tutorials out there mention [DevKit Advance](https://devkitadv.sourceforge.net/). This is a
+very outdated toolchain (the last release was in 2003!), don't use it! Use devkitARM instead, which
+is updated regularly.
+
+#### Assembly
+
+Even though you don't need to use assembly to develop for GBA, it is still an option, and you will
+need it if you're creating things like a 3D game, or an audio mixer.
+
+All of the C and C++ options also support assembly, so you can use them without any issues. 
+
+There is also [gvasm](https://github.com/velipso/gvasm) if you want an option that only focuses on
+assembly.
+
+#### Nim
+
+Check [Natu](https://github.com/exelotl/natu). It provides a library to write games and tooling to
+convert assets.
+
+#### Rust
+
+Check the [agb library](https://github.com/agbrs/agb).
+
+### Development from scratch
+
+If you don't want to use any libraries to create your game, it's possible to ignore all of them and
```

Comment:
Isn't this a bit the same of the assembly option?

Reply:
Hmmm maybe, yes. I guess that I'm trying to make different sections for:
- "I want to write absolutely every piece of code that will go into my product, interrupt handlers, etc"
- "I just want to be able to use assembly because I want to optimize my code, but I'm open to using pieces of code provided by other people".